collapse/setup-collapse.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+# install stable collapse
+mkdir -p ./collapse/r-collapse
+Rscript -e 'install.packages(c("collapse", "dplyr"), lib="./collapse/r-collapse")'
```

Comment:
can you change this to 
`Rscript -e 'install.packages(c("collapse", "dplyr", "Rcpp"), lib="./collapse/r-collapse", repos = "http://cloud.r-project.org")'`

That should get everything running

Reply:
I made the changes and asked @vincentarelbundock to update the PR. Hopefully that will solve the issue. dplyr was removed because it is no longer needed.